modal.js

```diff
@@ -0,0 +1,144 @@
+  myTechStack: ['Javascript', 'HTML', 'CSS', 'Github', 'Ruby', 'Bootstrap'],
+  liveDemoButton: 'https://joyapisi.github.io/Quote-Generator/',
+  sourceLinkButton: 'https://github.com/joyapisi/Quote-Generator',
+},
+{
+  id: 4,
+  title: 'Tonic',
+  jobDetails: ['CANOPY', 'BackEndDev', '2015'],
+  featuredimage: 'sources/Portfolio4.png',
+  jobDescriptionDetails: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem',
+  myTechStack: ['Javascript', 'HTML', 'CSS', 'Github', 'Ruby', 'Bootstrap'],
+  liveDemoButton: 'https://joyapisi.github.io/Quote-Generator/',
+  sourceLinkButton: 'https://github.com/joyapisi/Quote-Generator',
+}];
+
+function generateModal(
+  id,
+  title,
+  jobDetails,
+  featuredimage,
+  jobDescriptionDetails,
+  myTechStack,
+  liveDemoButton,
+  sourceLinkButton,
+) {
+  return `<section class="popupwindow" id="popup-project-${id}">
+  <div class="popup">
+    <div class="top-title">
+      <h2 class="title">${title}</h2>
+      <button id="closed-popup-${id}" class="closed-popup"> 
+      <img src="./sources/closeButton.png" alt=""> </button>  
+    </div>
+  
+    <ul class="job-title">
+      <li class="canopy-title">
+      ${jobDetails[0]}
+      </li>
+      <li class="dot-icon">
+        <img class="dot-icon-img" src="./sources/dot.png" alt="dot"/>
+        ${jobDetails[1]}
+      </li>
+      <li class="year-2015">
+        <img class="year-tonic" src="./sources/dot.png" alt="year-tonic"/>
+        ${jobDetails[2]}
+      </li>
+    </ul>
+
+  <img id="modal-image" src="${featuredimage}" alt="picture">
+
+
+   <div class="bottom-content">
+       <p class="popupparagraph2">${jobDescriptionDetails}</p>
+   
+       <div class="right-block">
+         <div class="techstackbuttons">
+           <ul class="popupresources" title="popup-resources">
+             <li><a  class="popupbutton" href="#">${myTechStack[0]}</a><li>
+             <li><a  class="popupbutton" href="#">${myTechStack[1]}</a></li>
+             <li><a  class="popupbutton" href="#">${myTechStack[2]}</a></li>           
+           </ul>
+         </div>
+         
+         <div class="popupaction">
+           <button class="popupseeLiveButton">
+             <a href="${liveDemoButton}"> 
+             See Live
+             <img src="./sources/popupaction.png" alt="see image live" />
+             </a> 
+             </button> 
+           <button class="popupseeSourceButton " > 
+           <a href="${sourceLinkButton}"> 
+           See Source 
+             <img src="./sources/popupgithub.png" alt="github source" />
+             </a>
+             </button>
+         </div>
+       </div>
+     </div>
+   </div>
+</section>
+  `;
+}
```

Comment:
- [ ] I can see you implemented some of the requested changes by the previous reviewer's good job 👍 but there I one you are missing on your pop-up as requested by the previous reviewer kindly consider adding the scroll on the pop-up instead of adding it on the main page like when the popup is open a user can scroll on it to view more information if you can see well on the below image am attaching you can see that I view half of the pop-up I can't scroll
-
![her](https://user-images.githubusercontent.com/60146030/224497690-15a07ef6-f832-430e-8b78-e76103d62cdf.PNG)
![gg](https://user-images.githubusercontent.com/60146030/224498498-6baf0416-30c2-4946-b8fe-7a3ad2e676f0.PNG)

Reply:
[OPTIONAL] Nice implementing the changes 👍🏽 but the design can be improved if you increase the height of the popup. This is what you have so far 👇🏽 

![Screenshot 2023-03-11 at 13 53 39](https://user-images.githubusercontent.com/45739963/224506522-17af424a-bbe9-4d3c-b618-2cd1e677c32d.png)

It looks good 💯 but I feel like it can have a greater height. Thoughts? 🤔